CMakeLists.txt

```diff
@@ -48,16 +48,78 @@ if ((FIPS_MACOS OR FIPS_WINDOWS OR FIPS_LINUX) AND NOT (FIPS_RASPBERRYPI OR FIPS
             fips_libs(opengl32)
         elseif (FIPS_LINUX)
             add_definitions(-DFIPS_GLFW_LINUX=1)
-            fips_files(
-                x11_platform.h xkb_unicode.h linux_joystick.h
-                posix_time.h posix_thread.h
-                x11_init.c x11_monitor.c x11_window.c
-                xkb_unicode.c linux_joystick.c posix_time.c posix_thread.c
-                glx_context.h glx_context.c
-                egl_context.h egl_context.c
-                osmesa_context.h osmesa_context.c
-            )
-            fips_libs(X11 Xrandr Xi Xinerama Xxf86vm Xcursor GL m ${CMAKE_DL_LIBS})
+            if (NOT DEFINED FIPS_WAYLAND)
+                set(FIPS_WAYLAND FALSE CACHE BOOL "Set to true to build for wayland" FORCE)
+            endif()
+            if (FIPS_WAYLAND)
+                add_definitions(-DFIPS_WAYLAND=1)
+                find_package(ECM REQUIRED NO_MODULE)
+                list(APPEND CMAKE_MODULE_PATH "${ECM_MODULE_PATH}" "${CMAKE_SOURCE_DIR}/cmake_modules")
```

Comment:
The ```"${CMAKE_SOURCE_DIR}/cmake_modules"``` doesn't make sense here and should be removed (CMAKE_SOURCE_DIR is the ```sokol_samples``` or ```oryol``` project directory at this point).

Instead I had to add the ```CMake/modules``` in the glfw project directory to make the following find_packages work, so this becomes:

```list(APPEND CMAKE_MODULE_PATH "${ECM_MODULE_PATH}" "${CMAKE_CURRENT_LIST_DIR}/glfw/CMake/modules")```

Reply:
Oh oops sorry - I'd copied the modules from GLFW into the root dir when I was first messing around trying to get it to work.